environment.yml

```diff
@@ -17,5 +19,6 @@ dependencies:
   - conda-forge::rioxarray=0.10.1
   - conda-forge::torchgeo=0.2.0
   - pip:
+    - deepspeed==0.6.4
```

Comment:
TODO wait for conda-forge package at https://github.com/conda-forge/staged-recipes/pull/19021 so this doesn't need to be installed from PyPI. Also check if using conda-forge package means `gcc`/`gxx_linux-64` isn't needed.